examples/dt/uni07eta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    ontap-iscsi:
+      customServiceConfig: |
+        [ontap]
+        volume_backend_name=ontap
+        volume_driver=cinder.volume.drivers.netapp.common.NetAppDriver
+        netapp_server_hostname=_replaced_
+        netapp_server_port=80
+        netapp_storage_protocol=iscsi
+        netapp_storage_family=ontap_cluster
+        consistencygroup_support=True
+      networkAttachments:
+        - storage
+      customServiceConfigSecrets:
+        - cinder-volume-ontap-secrets
+  cinder-volume-ontap-secrets: |
+    [ontap]
+    netapp_login = _replaced_
+    netapp_password = _replaced_
+    netapp_vserver = _replaced_
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift.SwiftBackupDriver
+    replicas: 1
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      debug=True
+      enabled_backends = default_backend:swift
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+
+    default:
+      replicas: 1
+
+  swift:
+    enabled: true
+
+
+  octavia:
+    enabled: false
```

Comment:
It is kept disabled in all other DTs at the moment.

It will be enabled after the following change is merged: https://github.com/openstack-k8s-operators/ci-framework/pull/1756/files

Reply:
ok i now see we have https://github.com/openstack-k8s-operators/architecture/commit/2a955bf6e13b01d94a11ab0998357429e85e5f34, would have been better to point to relevant issues whenever we disable things so context is at the same place of where it's disabled and avoid clarifications.